infrastructure/Makefile

```diff
@@ -0,0 +1,52 @@
+#!make
+# include .env
+# export
+
+tf_development = ./
+# tf_tests = ./Infrastructure/tests
+
+tf-validate:
+	@if terraform fmt -check -diff -recursive  $(tf_development); then \
+		echo "Terraform files all formatted correctly"; \
+	else \
+		echo "Formatting errors found, run tf-format to fix automatically"; \
+		exit 1; \
+	fi
+	@cd $(tf_development) && terraform init -backend=false && terraform validate
+
+tf-deploy: tf-format tf-validate
+	@cd $(tf_development) && \
+		terraform init && \
+		terraform apply -auto-approve -var-file=./environments/dev.tfvars -var-file=credentials.tfvars -lock=false
+
+tf-destroy:
+	@cd $(tf_development) && \
+		terraform destroy -var-file=./environments/dev.tfvars -lock=false
+
+tf-format:
+	terraform fmt -recursive $(tf_development)
+
```

Comment:
Worth a comment to indicate why this is commented out at the moment? For future humans?

Reply:
Done